index.js

```diff
@@ -123,8 +147,10 @@ class Vec3 {
     return dx * dx + dy * dy + dz * dz
   }
 
-  equals (other) {
-    return this.x === other.x && this.y === other.y && this.z === other.z
+  equals (other, error = Number.EPSILON) {
```

Comment:
this is a breaking change; it should default to 0

Reply:
fixed via 6f028d5ac5844f589719b2298740b8eebd4719c4